deploy/crds/cloudingress.managed.openshift.io_apischemes_crd.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: apischemes.cloudingress.managed.openshift.io
+  name: APISchemes.cloudingress.managed.openshift.io
```

Comment:
This is odd. Is a resource allowed to have uppercase in its name? None of the CRDs on my stage cluster do.

Reply:
The [k8s docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/) imply that "most' object names can't contain uppercase letters. Even if CRDs are technically an execption, none of the other CRDs I'm looking at have uppercase in their names.

Reply:
To get this back to lowercase, I needed to update the kubebuilder resource path from APISchems to apischems. Marking as resolved.